File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int [] indices = new int[2];

        for (int i = 0; i < nums.length; i++){
            for (int k = i + 1; k < nums.length; k++){
                if (nums[i] + nums[k] == target){
                    indices[0] = i;
                    indices[1] = k; 
                    break;
                }
            }
        }

        return indices;
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        left, right = 0, len(numbers) - 1
        result = []


        while left < right:
            twoSum = numbers[left] + numbers[right]
            if twoSum < target:
                left += 1
            elif twoSum > target:
                right -= 1
            else:
                break

        result.append(left + 1)
        result.append(right + 1)
        return result